test/runner/lib/buildTestUrl.js

```diff
@@ -0,0 +1,32 @@
+import { generateModuleId } from "./generateHash.js";
+
+export function buildTestUrl(
+	modules,
+	{ browserstack, jquery, jqueryMigrate, port, reportId }
+) {
+	if ( !port ) {
+		throw new Error( "No port specified." );
+	}
+
+	const query = new URLSearchParams();
+	for ( const module of modules ) {
+		query.append( "moduleId", generateModuleId( module ) );
+	}
+
+	if ( jquery ) {
```

Comment:
Let's support the `esm / esmodules` option pair here as in Core.

Reply:
See https://github.com/jquery/jquery-migrate/pull/503#discussion_r1530491173